README.md

```diff
@@ -143,6 +148,23 @@ The check method has the following API:
 
   // Check: sends a service check (DataDog only)
   client.check('service.up', client.CHECKS.OK, { hostname: 'host-1' }, ['foo', 'bar'])
+```
+
+### UDS (Unix Domain Socket) with Retry Support
```

Comment:
I like the addition of an example, but this cuts apart the existing examples and is at an odd place.  I suggest removing the headline and just making it another example shown at the end of the list of examples.